app/controllers/users/sessions_controller.rb

```diff
@@ -4,7 +4,14 @@ class Users::SessionsController < Devise::SessionsController
   class EmailNotFoundError < StandardError; end
   class LoginIncompleteError < StandardError; end
 
-  TEST_USERS = %w[admin@example.com early-career-teacher@example.com mentor@example.com].freeze
+  TEST_USERS = %w[
+    admin@example.com
+    early-career-teacher-1@example.com
```

Comment:
Let's include the name of the course in the email please - remembering which course which number corresponds to is a pain.